README.md

```diff
@@ -88,12 +88,12 @@ This is mainly used for debugging `latest` using the TEST and LIVE environment b
 To run TEST bundles on localhost:
 - In `envConfig/test.env` change the value of `LOG_DIR='/var/log/simorgh'` to `LOG_DIR='log'`
 - Then run `rm -rf build && npm run build:test && npm run start`
-- Visit a test article: http://localhost:7080/news/articles/c0g992jmmkko
+- Visit a test article: http://localhost.bbc.com:7080/news/articles/c0g992jmmkko
 
 To run LIVE bundles on localhost:
 - In `envConfig/live.env` change the value of `LOG_DIR='/var/log/simorgh'` to `LOG_DIR='log'`
 - Then run `rm -rf build && npm run build:live && npm run start`
-- Visit a live article: http://localhost:7080/news/articles/c8xxl4l3dzeo
+- Visit a live article: http://localhost.bbc.com:7080/news/articles/c8xxl4l3dzeo
 
 ## Changing request location
 
```

Comment:
After discussing with @dr3 - we should move the `/etc/hosts` information below to the `## Local Development` section of the Readme above, so that people new to cloning the application will have the instructions at hand to ensure the `localhost.bbc.com` endpoint will work for them.